homeassistant/components/meteo_france/sensor.py

```diff
@@ -177,18 +180,37 @@ class MeteoFranceRainSensor(MeteoFranceSensor):
     @property
     def state(self):
         """Return the state."""
-        return self.coordinator.data.forecast[0]["desc"]
+        return (
+            dt_util.as_local(self.coordinator.data.next_rain_date_locale())
+            if self.coordinator.data.next_rain_date_locale() is not None
+            else None
+        )
 
     @property
     def device_state_attributes(self):
         """Return the state attributes."""
+        next_rain_datetime = (
+            dt_util.as_local(self.coordinator.data.next_rain_date_locale())
+            if self.coordinator.data.next_rain_date_locale() is not None
+            else None
+        )
+        if next_rain_datetime:
+            rain_text_summary = (
+                f"La pluie est attendue à {next_rain_datetime.strftime('%H:%M')}."
+            )
+        else:
+            rain_text_summary = "Pas de pluie dans la prochaine heure."
+
         return {
-            **{
-                datetime.fromtimestamp(item["dt"]).strftime("%Y-%m-%d %H:%M:%S"): item[
-                    "desc"
-                ]
+            "1 hour forecast": [
```

Comment:
I think it needs to be underscored to handle the key properly for automations (the UI don't show it)

Reply:
done